script/genkeys

```diff
@@ -13,13 +13,23 @@ KEYMASTER="docker run --volume ${ROOT}/certificates:/certificates --rm cloudpipe
 if [ ! -f "${ROOT}/certificates/password" ]; then
   echo ">> creating a random password in ${ROOT}/certificates/password."
   touch ${ROOT}/certificates/password
-  chmod 600 ${ROOT}/certificates/password
+  chmod 640 ${ROOT}/certificates/password
   # "If the same pathname argument is supplied to -passin and -passout arguments then the first
   # line will be used for the input password and the next line for the output password."
-  cat /dev/random | head -c 128 | base64 > ${ROOT}/certificates/password
+  cat /dev/urandom | head -c 128 | base64 > ${ROOT}/certificates/password
```

Comment:
We'd prefer to keep `/dev/random`, even though it's blocking. I'm open to discussion about this piece, but I imagine it will go [like this](http://www.2uo.de/myths-about-urandom/).